src/lang/base/BuiltIns.mli

```diff
@@ -34,7 +34,7 @@ module ScillaBuiltIns
      * Executor for evaluating the operation      
     *)
     val find_builtin_op :
-      ER.rep ident -> typ list -> ((typ * typ * built_in_executor), scilla_error list) result
+      builtin -> typ list -> loc -> ((typ * typ * built_in_executor), scilla_error list) result
```

Comment:
Once you introduce the type `builtin_ident` that @anton-trunov suggested, this function should no longer accept a `loc` as a parameter.